temporalio/testing/_workflow.py

```diff
@@ -91,6 +91,7 @@ async def start_local(
         download_dest_dir: Optional[str] = None,
         ui: bool = False,
         runtime: Optional[temporalio.runtime.Runtime] = None,
+        search_attributes: Sequence[temporalio.common.SearchAttributeKey] = [],
```

Comment:
This and existing parameters are violating no-mutable-defaults.

Reply:
I am not aware of this rule on our codebase. We don't mutate this parameter but Python is not expressive enough to mark a list as immutable. It is an immutable default despite Python's lack of expressiveness. I wouldn't expect such a rule to affect whether the PR is approved.

Reply:
It's a standard rule in Python that default argument values shouldn't be mutable, e.g.

https://google.github.io/styleguide/pyguide.html#2124-decision
https://docs.astral.sh/ruff/rules/mutable-argument-default/

We can fix the existing examples in a separate PR, but let's not add more violations in this PR.

Reply:
I disagree, I think we should be consistent with what's there and not institute new rules after PRs are opened

Reply:
OK, I mean, you'd only have to change it from `[]` to `()` 😄  But I agree we don't want to confuse this PR with purging the codebase of all existing violations. I've never worked in a Python codebase that doesn't enforce that rule. Opened https://github.com/temporalio/sdk-python/issues/762

Reply:
Ah, that is simple enough, I will make that change. I do wish the rule was to not mutate a parameter with a mutable default instead of just not having a mutable default.

Reply:
Fixed